src/components/account/account.test.tsx

```diff
@@ -108,36 +110,36 @@ describe('account test suite', () => {
 
         const response = await account.getGoogleOIDCCallback(ctx, params);
         expect(response.body).toContain('try again later');
+        expect(testSpacing(response.body as string)).toHaveLength(0);
```

Comment:
Ditto with above. I'm guessing this is possible then. It may help to rename `testSpacing` to describe its behaviour.

P.S., I'm OK with this being done in a followup PR. 🤗